neo/Network/P2P/TaskManager.cs

```diff
@@ -265,6 +281,23 @@ private void RequestTasks(TaskSession session)
             {
                 session.RemoteNode.Tell(Message.Create("ping", PingPayload.Create(Blockchain.Singleton.Height)));
             }
+            if (!HasStateRootTask)
+            {
+                var state_height = Math.Max(Blockchain.Singleton.StateHeight, ProtocolSettings.Default.StateRootEnableIndex - 1);
+                var height = Blockchain.Singleton.Height;
+                if (state_height + 1 < height)
+                {
+                    var state = Blockchain.Singleton.GetStateRoot(state_height + 1);
+                    if (state.Flag == StateRootVerifyFlag.Unverified)
```

Comment:
`state` could be `null`
```suggestion
                    if (state == null || state.Flag == StateRootVerifyFlag.Unverified)
```